cli/template.hbr

```diff
@@ -0,0 +1,28 @@
+version: "3.9"
```

Comment:
This file is not a handlebars template, or is it?

Reply:
it is not, renamed